README.md

```diff
@@ -160,6 +160,7 @@ Flags:
   -b, --baseline-path          path to a previously generated report with known issues that gitleaks should ignore
   -s, --source string          path to source (git repo, directory, file)
   -v, --verbose                show verbose output from scan
+  --max-target-megabytes       maximum size for a file/blob to be scanned
```

Comment:
I suggest adding something like "Files larger than this will be skipped". Just to be totally explicit about the behavior.